webpack/stripes-module-parser.js

```diff
@@ -111,10 +111,11 @@ class StripesModuleParser {
   // This will be modified to a webpack-compatible dynamic import when we implement code-splitting.
```

Comment:
In fact, hasn't this now been modified into a webpack-compatible dynamic import? 🎉 👏